corehq/apps/users/views/mobile/users.py

```diff
@@ -671,6 +671,9 @@ def get_pagination_data(self, in_data):
 
         # backend pages start at 0
         users_query = self._user_query(query, page - 1, limit)
+
+        # run with a blank query to fetch total records with same scope as in search
+        total_records = self._user_query('', 0, 0).run().total
```

Comment:
Thank You @benrudolph. Did set offset and limit to 0 to ensure no results. But this definitely sounds the better way.